docker-compose.yml

```diff
@@ -18,11 +18,12 @@ services:
       - media_volume:/experimenter/experimenter/media
       - /experimenter/experimenter/legacy/legacy-ui/core/.cache/
       - /experimenter/experimenter/legacy/legacy-ui/core/node_modules/
+      - /experimenter/experimenter/theme/static_src/node_modules/
       - /experimenter/experimenter/nimbus-ui/node_modules/
       - /experimenter/experimenter/served/
       - /experimenter/node_modules/
       - ${GOOGLE_ADC_FILE}:${GOOGLE_APPLICATION_CREDENTIALS}:ro
-    command: bash -c "python bin/setup_kinto.py;/experimenter/bin/wait-for-it.sh db:5432 -- python /experimenter/manage.py runserver 0:7001"
+    command: bash -c "python bin/setup_kinto.py;/experimenter/bin/wait-for-it.sh db:5432 -- python /experimenter/manage.py runserver 0:7001 & yarn --cwd /experimenter/experimenter/theme/static_src run dev"
```

Comment:
Actually, one final nit:

Can we rewrite this with the yarn cmd running in the background perhaps?

Reply:
```
    command: bash -c "python bin/setup_kinto.py;/experimenter/bin/wait-for-it.sh db:5432 --  yarn --cwd /experimenter/experimenter/theme/static_src run dev &; python /experimenter/manage.py runserver 0:7001"
```

Reply:
see my other comment